### PR TITLE
[FIX] theme_*: enable footer/header template correctly

### PR DESCRIPTION
**Pull request by rdeodoo**
Community: https://github.com/odoo/odoo/pull/81181
Themes: https://github.com/odoo/design-themes/pull/536
Upgrade: https://github.com/odoo/upgrade/pull/3107

### CODE REVIEW

File: theme_anelusia/models/theme_anelusia.py [hunk 1]
```diff
@@ -5,8 +5,4 @@ class ThemeAnelusia(models.AbstractModel):
     _inherit = 'theme.utils'
 
     def _theme_anelusia_post_copy(self, mod):
-        self.disable_view('website.footer_custom')
         self.enable_view('website.template_footer_headline')
-
-        # For compatibility
-        # self.enable_asset('theme_common.compatibility_saas_10_1')
```
Comment (qsm-odoo) [inline theme_anelusia/models/theme_anelusia.py:12]: Mmh, ok

**Review by qsm-odoo — APPROVED** [review]
14.0 too though? :)

**Comment by rdeodoo** [comment]
About 14.0 or 15.0, we will see what is the conclusion on the other PR I guess :)

**Comment by rdeodoo** [comment]
@robodoo r+

**Pull request closed without merging**